### PR TITLE
Add moblie first

### PR DESCRIPTION
**Pull request by Blessman-Newton**
**Task: Portfolio**

**In this PR, We have added footer and header to the page**

`Added the toolbar`
`Added the headline section`
`Implemented the button interactions`
`Used HTML & CSS `

_**I have ensured that:**_

1. There are no linter errors in the project.
2. Used Correct git-flow.
3.  Included a descriptive Readme file.
4. Used descriptive commit messages.
5. Followed the recommended HTML and CSS.
6. Documented my work in a professional manner according to Professional code
7.  Add node_modules to gitignore file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    <path fill-rule="evenodd" d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"/>
+                </svg>
+            </div>
+        </nav>
+    </header>
+    <section>
+        <div class="container">
+            <div class="row">
+                <h1>Hey There. <br> I'm Bless.</h1>
+                <h2>I'm a Software Developer</h2>
```
Comment (bobb-Rob) [inline index.html:29]: - [ ] This text `<h2>I'm a Software Developer</h2>` should use the font family of `Roboto` according to the figma design. Please fix this.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                <p class="text">
+                    I can help you build a product , feature
+                    or website Look through some of my 
+                    work and experience! If you like what you 
+                    see and have a project you need coded, 
+                    don’t hesitate to contact me.
+                </p>
```
Comment (bobb-Rob) [inline index.html:36]: - [ ] This text `<p class="text"> I can help you build a product, feature... </p>` should use the font family of `poppins` according to the Figma design. Please fix this.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                        <li><a href="https://twitter.com/BlessmanNewton"><i class="fa fa-twitter" aria-hidden="true"></i></i></a></li>
+                        <li><a href="https://medium.com/BlessmanNewton"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+                    </ul>
+                </div>       
+            </div>
+            </div>
+        </div>
+        <div class="shapels">
+            <img src="imakges/Illustration (1).png" alt="">
+        </div>
```
Comment (bobb-Rob) [inline index.html]: - [ ] If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Blessman-Newton , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @Blessman-Newton, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All required changes fixed ✔️
- Commit messages meaningful ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**